apps/angular/src/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable, inject } from '@angular/core';
+
+import { AnimeDto } from '../dtos/anime.dto';
+import { Anime } from '../models/anime';
+
+import { AnimeStatusMapper } from './animeStatus.mapper';
+import { AnimeTypeMapper } from './animeType.mapper';
+
+/** Anime dto transform service. */
```

Comment:
It can possibly map many things, not only DTO
```suggestion
/** Anime mapper. */
```